lib/src/google_fonts_base.dart

```diff
@@ -221,3 +237,43 @@ int _computeMatch(GoogleFontsVariant a, GoogleFontsVariant b) {
   }
   return score;
 }
+
+Future<Map<String, dynamic>> _loadAssetManifestJson() async {
+  try {
+    final jsonString = await rootBundle.loadString('AssetManifest.json');
+    return json.decode(jsonString) as Map<String, dynamic>;
+  } catch (e) {
+    rootBundle.evict('AssetManifest.json');
+    return null;
+  }
+}
+
+bool _isFamilyWithVariantInManifest(
+  GoogleFontsFamilyWithVariant familyWithVariant,
+  Map<String, dynamic> manifestJson,
+) {
+  if (manifestJson == null) return false;
+
+  for (final assetList in manifestJson.values) {
+    for (final String asset in assetList) {
+      String matchingFontSuffix;
+      for (final suffix in ['.ttf', '.otf']) {
```

Comment:
We should double check which font files are supported by fonts.google.com.

Reply:
I made a ticket for it